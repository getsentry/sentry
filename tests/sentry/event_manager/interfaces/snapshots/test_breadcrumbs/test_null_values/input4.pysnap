---
created: '2019-05-22T14:45:11.059085Z'
creator: sentry
source: tests/sentry/event_manager/interfaces/test_breadcrumbs.py
---
errors: null
to_json: null
