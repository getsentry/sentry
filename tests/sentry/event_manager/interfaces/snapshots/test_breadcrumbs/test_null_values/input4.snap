errors: null
to_json: {}
